Database/Stored Procedures/Add_or_Update/add_or_update_visitor.sql

```diff
@@ -81,30 +125,58 @@ GO
 
 -- Test INSERT
 BEGIN TRAN
-EXEC sp_add_or_update_visitor NULL, 1, 'yuriz@live.nl', 'Yuri', 'Vannisselroy', '0682006373', '1996-09-25', 'NULL', 'NULL', 'NULL', 'NULL', 1
+EXEC sp_add_or_update_visitor NULL, 'yuriz@live.nl', 'Yuri', 'Vannisselroy', '0682006373', '1996-09-25', 'NULL', 'NULL', 'NULL', 'NULL', 'Nijmegen', 'Nederland', 1
+ROLLBACK TRAN
+GO
+
+BEGIN TRAN
+EXEC sp_add_or_update_visitor NULL, 'yuriz@live.nl', 'Yuri', 'Vannisselroy', '0682006373', '1996-09-25', 'NULL', 'NULL', 'NULL', 'NULL', NULL, NULL, 1
+ROLLBACK TRAN
+GO
+
+-- Test INSERT (new town)
+BEGIN TRAN
+EXEC sp_add_or_update_visitor NULL, 'yuriz@live.nl', 'Yuri', 'Vannisselroy', '0682006373', '1996-09-25', 'NULL', 'NULL', 'NULL', 'NULL', 'Amstelveen', 'Nederland', 1
+ROLLBACK TRAN
+GO
+
+-- Test INSERT (new town and country)
+BEGIN TRAN
+EXEC sp_add_or_update_visitor NULL, 'yuriz@live.nl', 'Yuri', 'Vannisselroy', '0682006373', '1996-09-25', 'NULL', 'NULL', 'NULL', 'NULL', 'Madrid', 'Spanje', 1
 ROLLBACK TRAN
 GO
 
 -- Test INSERT (cannot insert duplicate email)
 BEGIN TRAN
-EXEC sp_add_or_update_visitor NULL, 1, 'robertverkerk@hetnet.nl', 'Yuri', 'Vannisselroy', '0682006373', '1996-09-25', 'NULL', 'NULL', 'NULL', 'NULL', 1
+EXEC sp_add_or_update_visitor NULL, 'robertverkerk@hetnet.nl', 'Yuri', 'Vannisselroy', '0682006373', '1996-09-25', 'NULL', 'NULL', 'NULL', 'NULL', 'Amstelveen', 'Nederland', 1
+ROLLBACK TRAN
+GO
+
+-- Test INSERT (must have both town and country either NULL or NOT NULL)
+BEGIN TRAN
+EXEC sp_add_or_update_visitor NULL, 'yuriz@live.nl', 'Yuri', 'Vannisselroy', '0682006373', '1996-09-25', 'NULL', 'NULL', 'NULL', 'NULL', NULL, 'Nederland', 1
+ROLLBACK TRAN
+GO
+
+BEGIN TRAN
```

Comment:
hier geen comment neergezet over wat getest wordt

Reply:
omdat het dezelfde test is als die erboven